examples/computecloth/computecloth.cpp

```diff
@@ -365,13 +399,14 @@ class VulkanExample : public VulkanExampleBase
 			storageBufferSize,
 			particleBuffer.data());
 
-		// SSBOs will be used both as storage buffers (compute) and vertex buffers (graphics)
+		// storageBuffers.input is used as an SSBO and only on the compute queue as an intermediate location for iterative computation
 		vulkanDevice->createBuffer(
-			VK_BUFFER_USAGE_VERTEX_BUFFER_BIT | VK_BUFFER_USAGE_STORAGE_BUFFER_BIT | VK_BUFFER_USAGE_TRANSFER_DST_BIT,
+			VK_BUFFER_USAGE_STORAGE_BUFFER_BIT | VK_BUFFER_USAGE_TRANSFER_DST_BIT | VK_BUFFER_USAGE_TRANSFER_SRC_BIT,
```

Comment:
As noted, `storageBuffers.input` is never accessed on the graphics queue, so no need for the vertex buffer bit, but I do need to copy from it to the output buffer in the compute queue, so added the transfer src bit.